game.py

```diff
@@ -545,7 +588,7 @@ def handleMessage(self, player, message):
             req_maze
                 Request to please send me the maze.  Reply is maze:.
 
-            maze: running_time, seed, width, height
+            maze: running_time, seed, width, height, hole
```

Comment:
"hole" is not defined here.

Reply:
@quozl 
I have corrected it and changed the variable name to `add_hole`.